modules/installation-options.adoc

```diff
@@ -1,9 +1,10 @@
 // Module included in the following assemblies:
 //
 // * architecture/architecture.adoc
+//* architecture/installation-options.adoc
 
 [id="installation-options-{context}"]
```

Comment:
Would it make sense to move the file to one called `installation-and-update.adoc` and update all references now that the title has changed?